packages/dcos-signal/extra/dcos-signal-ee.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=DC/OS Signal: reports cluster telemetry and analytics to help improve DC/OS
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartSec=20
+LimitNOFILE=16384
+PermissionsStartOnly=True
+User=dcos_signal
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/cfn_signal_metadata
+ExecStartPre=/bin/ping -c1 leader.mesos
+ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-signal
+
+# Signal depends on the dcos-licensing service for the license_id
+ExecStart=/bin/sh -c '[[ $DCOS_VARIANT == "enterprise" ]] && /opt/mesosphere/bin/dcos-signal -dcos-variant $DCOS_VARIANT -segment-key 51ybGTeFEFU1xo6u10XMDrr6kATFyRyh -licensing-socket /run/dcos/dcos-licensing.sock || true'
```

Comment:
I don't like idea of having ee specific code in oss. The only diference is in `DCOS_VARIANT` can we handle it diferently, e.g., set `DCOS_VARIANT` in `/opt/mesosphere/envirnment`?

Reply:
It **is** set in the `environment` file and referenced here. The idea behind this is that this service unit only runs on EE clusters. Otherwise we would have to have an EE package only for the service files which I don't find convenient in this case.